include/core/mir/geometry/displacement.h

```diff
@@ -91,6 +91,11 @@ inline constexpr Displacement operator-(Point const& lhs, Point const& rhs)
     return Displacement{lhs.x - rhs.x, lhs.y - rhs.y};
 }
 
+inline constexpr Displacement operator-(Size const& lhs, Size const& rhs)
+{
+    return Displacement{lhs.width - rhs.width, lhs.height - rhs.height};
+}
```

Comment:
Sorry, should have commented on this one too: it is wrong. (As discussed in #1064)

Reply:
Good catch